pkg/arguments/arguments.go

```diff
@@ -204,6 +204,14 @@ func CheckIgnoredAutoscalingFlags(autoscaling cluster.Autoscaling, computeNodes
 		if computeNodes != 0 {
 			return fmt.Errorf("--compute-nodes is meaningless with --enable-autoscaling")
 		}
+
+		if autoscaling.MinReplicas == 0 {
+			return fmt.Errorf("--min-replicas flag is required with --enable-autoscaling")
+		}
+
+		if autoscaling.MaxReplicas == 0 {
+			return fmt.Errorf("--max-replicas flag is required with --enable-autoscaling")
```

Comment:
consider renaming `CheckIgnoredAutoscalingFlags` to `CheckAutoscalingFlags`.
The "CheckIgnored..." functions originally were about flag combinations where otherwise one of them would be ignored.

Reply:
Agree

Reply:
fixed